apps/finance/app/src/components/TransferRow.js

```diff
@@ -121,6 +129,14 @@ class TransferRow extends React.PureComponent {
   }
 }
 
+const StyledTableRow = styled(TableRow)`
+  td:first-child {
```

Comment:
Similar to https://github.com/aragon/aragon-apps/pull/722#discussion_r270040540.